src/type-extensions.ts

```diff
@@ -0,0 +1,26 @@
+import "hardhat/types/config";
+
+declare module "hardhat/types/config" {
+  export interface HttpNetworkUserConfig {
+    privateKey?: string;
```

Comment:
define sth like `GodwokenUserConfig` and have here:
```
  godwoken: GodwokenUserConfig
```

Reply:
Zapiszę to jako polyjuiceConfig